pkg/sentry/fsimpl/devpts/terminal.go

```diff
@@ -49,6 +53,8 @@ func newTerminal(n uint32) *Terminal {
 		masterKTTY:  &kernel.TTY{Index: n},
 		replicaKTTY: &kernel.TTY{Index: n},
 	}
+
+	t.ld.SetTerminal(&t)
```

Comment:
We don't need the SetTerminal method since lineDiscipline is in this package.  Just set t.ld.terminal directly.